cider/exec/template/NativeCodegen.cpp

```diff
@@ -1491,7 +1503,14 @@ Executor::compileWorkUnit(const std::vector<InputTableInfo>& query_infos,
   LOG(ASM) << "CODEGEN #" << counter << ":";
 #endif
 
-  nukeOldState(allow_lazy_fetch, query_infos, &ra_exe_unit);
+  //  nukeOldState(allow_lazy_fetch, query_infos, &ra_exe_unit);
```

Comment:
remove it directly?

Reply:
we fetch executor_ from a pool by now, so it's been reused. all members in executor(include cgen_state_) should be reset(clean). you should comment on this line before a local executor is used.

Reply:
CgenStateManager will reset CgenState instances